src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1498,6 +1592,8 @@ private void DoOrganellePlaceAction(EditorAction action)
             organelle.Position);
 
         editedMicrobeOrganelles.Add(organelle);
+
+        HandleIslands();
```

Comment:
All of these HandleIslands calls should be removed.
Just looked up that there is `private void OnOrganellesChanged()`
which has all the required handling logic for when an organelle is added or removed.

Reply:
The problem with the OnOrganellesChanged is that it gets called multiple times when working in symmetry mode.
The correct order is:
- Add all organelles
- Handle Islands

Reply:
But shouldn't the code be made that so that even if it gets called a bunch of times (which is something I do kinda want to improve in the future), it eventually results in the right result?
The code would each time compute the islands and assign materials. Then only the final result of the last OnOrganellesChanged call would be visible.

Reply:
Ah yeah, a small list clear did the job.
Calling OnOrganellesChanged would probably fix performance for big species

Reply:
So far it has seemed good enough, but I'll open an issue: https://github.com/Revolutionary-Games/Thrive/issues/1723